cypress/e2e/assignment/login.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('login', () => {
+  it('can login and access profile', () => {
+    cy.clock();
+    cy.visit('/');
+    cy.tick(1000);
+    cy.get('#registerForm').within(() => {
+      cy.get('button[data-bs-target="#loginModal"]').click();
+    });
+    cy.tick(1000);
+    cy.get('#loginEmail').type('TEST_EMAIL', { delay: 500 });
+    cy.get('#loginPassword').type('TEST_PASSWORD{enter}', { delay: 500 });
```

Comment:
How do you know it worked? You could check localstorage

Reply:
Done checking on localstorage